pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1839,7 +1841,7 @@ func (ctrl *PersistentVolumeController) provisionClaimOperationExternal(
 		return provisionerName, err
 	}
 	claim = newClaim
-	msg := fmt.Sprintf("waiting for a volume to be created, either by external provisioner %q or manually created by system administrator", provisionerName)
+	msg := fmt.Sprintf("Requested external provisioner %q for a volume to be created, or waiting manually created volume by system administrator", provisionerName)
```

Comment:
Agreed. I like
>  "Waiting for a volume to be created (either by the external provisioner %q, or manually by the system administrator)"

Just to go one step further...
" Waiting for a volume to be created (either by the external provisioner %q, or manually by the system administrator). Please ensure that the provisioner exists and has been properly registered."